.nix/coq-overlays/mathcomp-single/default.nix

```diff
@@ -11,4 +11,12 @@
   propagatedBuildInputs = old.propagatedBuildInputs ++
                           [ coq-elpi hierarchy-builder ];
   installPhase = "echo NO INSTALL";
+  doCheck = true;
+  checkPhase = ''
+    make -C mathcomp clean
+    COQ_ELPI_ATTRIBUTES=log_hb make -j $(max_jobs) -C mathcomp ./algebra/ssralg.vo
+    hb patch `find . -name \*.v`
+    sed -i 's/Notation EqMixin := Equality.Mixin./Notation EqMixin := (Equality.Mixin _)./' ssreflect/eqtype.v
```

Comment:
????

Reply:
Some changes in the implementation of implicits?

Reply:
I think it's a bug, either of Coq's pretty printer. Given it only affects this thing, I hacked it to move on and see...